Ductus.FluentDocker/Services/Extensions/ContainerExtensions.cs

```diff
@@ -280,6 +280,45 @@ public static IContainerService WaitForHealthy(this IContainerService service, l
 
 			return service;
 		}
+		
+		/// <summary>
+		///   Waits for a specific message in the logs
+		/// </summary>
+		/// <param name="service">The service to check processes within.</param>
+		/// <param name="message">The message to wait for</param>
+		/// <param name="millisTimeout">Timeout giving up the wait.</param>
+		/// <returns>The inparam service.</returns>
+		public static IContainerService WaitForMessageInLogs(this IContainerService service, string message, long millisTimeout = -1)
+		{
+			if (service == null)
+				return null;
+
+			Exception exception = null;
+			var stopwatch = Stopwatch.StartNew();
+			using (var mre = new ManualResetEventSlim())
+			{
+				Timer timer;
+				using (timer = new Timer(_ =>
+				{
+					var logs = service.Logs().Read();
```

Comment:
Perhaps an improvement in a separate PR is to allow reading only partial logs using tail `100` as default but allow users to override the default tails length.

Reply:
@casz, really nice suggestion! As for the next PR :)